microsoft/botframework-solutions
# [BotBuilder Solutions][TypeScript] Implement tests

## Description
- Implement the following tests
  - `speechUtilityTest`
  - `responseGenerationTest`
  - `listExtensionsTest`
  - `dateTimeExtensionsTest`
- Replace `dateformat` with [`dayjs`](https://github.com/iamkun/dayjs) library
- Add a **customize zh locale** to be used by `dayjs` library
- Update **test folder structure**
- Update `npm-shrinkwrap.json` file


## Testing Steps

1. Go to `botframework-solutions\lib\typescript\`
2. Run `rush update`
3. Run `run rebuild`
4. Go to `botframework-solutions\lib\typescript\botbuilder-solutions`
5. Run `npm run test` and check the following output

![image](https://user-images.githubusercontent.com/11904023/58492333-16ff4500-8147-11e9-8b1d-25db57dd1fa8.png)

## Checklist
